### PR TITLE
[fk] update docs

### PR DESCRIPTION
**Pull request by witchfelicia**
# Summary
Add installation, getting started, usage, and community-building section in docs

# Tests
Uhh i did not 😳

cc: @wangxiaoyou1993

### CODE REVIEW

**Comment by wangxiaoyou1993** [comment]
Mage logo doesn't load when I preview the doc 
image

**Comment by witchfelicia** [comment]
I'm not sure how to fix ): it works on mine and idk why
image

**Comment by witchfelicia** [comment]
omg i get it! the logo is white 🥴

**Comment by wangxiaoyou1993** [comment]
> omg i get it! the logo is white 🥴

haha, that's funny.

**Comment by wangxiaoyou1993** [comment]
The usage section can be polished and screenshots can be updated once product is ready. For example, we should talk about:
* Edit and apply action in suggestions
* View column specific reports, visualizations and suggestions.

File: README.md [hunk 1]
```diff
@@ -1,12 +1,24 @@
...
 1. [Getting Started](#getting-started)
+2. [Usage](#usage)
+3. [To Contribute](#to-contribute)
 
 ## Introduction
-Mage AI is a Python library for data cleaning.
+Mage AI aims to simplify data pre-processing. With a few lines of code, you can:
+- Generate data cleaning suggestions in your local notebook
+- Display feature-specific statistics and visualizations
+- Easily detect "holes" and anomalies in your data
```
Comment (wangxiaoyou1993) [inline README.md:16]: Create data cleaning pipeline and clean your data with a few clicks.

File: README.md [hunk 2]
```diff
@@ -1,12 +1,24 @@
...
+
+Coming soon:
+- Access faster data analysis and cleaning using Mage's public cloud server
+- Perform no-code data transformations within the UI
+- Export cleaned data into your ML pipeline
```
Comment (wangxiaoyou1993) [inline README.md:21]: Support user defined cleaning functions within the UI.

Comment (wangxiaoyou1993) [reply README.md:21]: Update "Export cleaned data into your ML pipeline" to "Export data cleaning pipeline and integrate into your production data pipelines."

**Review by wangxiaoyou1993 — APPROVED** [review]

File: README.md (hunk 1 above)
Comment (witchfelicia) [reply README.md]: Xiaoyou, are you suggesting I replace lines 13 -16 with a single sentence "Create data cleaning pipeline and clean your data with a few clicks." or adding that sentence as a list item?

**Pull request merged**